manifests/server/puppetserver.pp

```diff
@@ -271,4 +272,11 @@
     ensure  => 'file',
     content => template('puppet/server/puppetserver/conf.d/metrics.conf.erb'),
   }
+
+  if $jolokia_metrics_policy != undef {
+    file { '/etc/puppetlabs/puppetserver/jolokia-access.xml' :
+      ensure => file,
```

Comment:
Does it make sense to ensure it's `absent` when `$jolokia_metrics_policy` is `undef`?

Reply:
Sorry for the super late reply, but I'd argue the absence of `$jolokia_metrics_policy` should not dictate enforcing the absence of the file. 

I could see ensuring `jolokia-access.xml` as absent if $jolokia_metrics_policy == `absent`, but not undef.